src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String WHITESPACE = " ";
+    private static final String COMA = ",";
+    private static final int INDEX_ZERO = 0;
+    private static final int INDEX_ONE = 1;
+    private static final int INDEX_TWO = 2;
+    private static final int ZERO = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] readedData = readFile(fromFileName);
+        String[] readyReport = generateReport(readedData);
+        writeToFile(readyReport, toFileName);
+    }
+
+    public String[] readFile(String fromFileName) {
```

Comment:
Consider changing method visibility to private, as it's not used outside of the class.